.editorconfig

```diff
@@ -0,0 +1,6 @@
+
+root = true
+
+[*.{js,ts}]
+indent_style = space
+indent_size = 2
```

Comment:
I'm aware, but we don't need to be establishing those kinds of rules right now. If anything, we can enforce this at the eslint level, but I don't think that's necessary. This is ancillary to this PR anyways.